### PR TITLE
docs: translate the dev guide

### PR DESCRIPTION
**Pull request by SoonIter**
## Summary

image

## Checklist

- [x] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/en/contribute/development/debugging.md [hunk 1]
```diff
@@ -108,19 +108,19 @@ npm run build:binding:debug
 npm run build:js
 ```
 
-### Configure `launch.json` in vscode
+### Configure `launch.json` in VSCode
```
Comment (chenjiahan) [inline website/docs/en/contribute/development/debugging.md:111]:
```suggestion
### Configure `launch.json` in VS Code
```

File: website/docs/zh/contribute/development/debugging.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
+# Debugging
+
+## 通过 VSCode 调试
```
Comment (SoonIter) [inline website/docs/zh/contribute/development/debugging.md:3]:
```suggestion
## 通过 VS Code 调试
```

File: website/docs/zh/contribute/development/prerequisites.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# 准备工作
+
+Rspack 使用 [Rust](https://rust-lang.org/) 和 [NAPI-RS](https://napi.rs/) 构建，然后被发布为 [Node.js](https://nodejs.org/) 包.
+
+## 安装 Rust
+
+- 使用 [rustup](https://rustup.rs/) 安装 Rust.
```
Comment (chenjiahan) [inline website/docs/zh/contribute/development/prerequisites.md:7]:
```suggestion
- 使用 [rustup](https://rustup.rs/) 安装 Rust。
```

File: website/docs/zh/contribute/development/prerequisites.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- 如果你在使用 VSCode，我们推荐安装 [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer) 扩展.
```
Comment (chenjiahan) [inline website/docs/zh/contribute/development/prerequisites.md]:
```suggestion
- 如果你在使用 VS Code，我们推荐安装 [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer) 扩展。
```

File: website/docs/zh/contribute/development/debugging.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
+# Debugging
+
+## 通过 VS Code 调试
+
+1. 安装 `go install github.com/go-delve/delve/cmd/dlv@latest`
+2. 安装 VSCode 扩展 [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer) 和 [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)
```
Comment (chenjiahan) [inline website/docs/zh/contribute/development/debugging.md]:
```suggestion
2. 安装 VS Code 扩展 [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer) 和 [CodeLLDB](https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb)
```

File: website/docs/zh/contribute/development/profiling.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
+# Profiling
+
+在本节中，我们将探讨如何分析 Rspack Profile 以识别性能瓶颈。
+
+通过检查 Rspack 将时间花在哪里，我们可以深入了解如何提高性能。
+
+由于不同的分析器有不同的优势。最好使用多个。
```
Comment (chenjiahan) [inline website/docs/zh/contribute/development/profiling.md:7]:
```suggestion
由于不同的分析器有不同的优势，使用多个分析器是一个好的选择。
```

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
Nice 👍

**Pull request merged**